src/Integration/Helpers/Constants.cs

```diff
@@ -60,7 +60,8 @@ public static class Constants
         /// <summary>
         /// SonarLint issues home page
         /// </summary>
-        public const string SonarLintIssuesWebUrl = "https://groups.google.com/forum/#!forum/sonarlint";
```

Comment:
I didn't realise we had this generic error message with an out-of-date link. However, it was only used in one place.

I've updated the link and added some line breaks in the message to make it easier to read.

Reply:
I had no idea we had this...